app/views/work_packages/_two_column_attributes.html.erb

```diff
@@ -29,12 +29,12 @@ See doc/COPYRIGHT.rdoc for more details.
 <% left_attributes, right_attributes = attributes.in_groups(2, false) %>
 
 <div class="grid-block wrap">
-  <div class="form--column">
+  <div class="form--column medium-6">
     <% left_attributes.each do |attribute| %>
       <%= attribute.field %>
     <% end %>
   </div>
-  <div class="form--column">
+  <div class="form--column medium-6">
```

Comment:
This is necessary as the current version of foundation is broken in regards to the grid it provides. We have a couple of places where we had to fix it this way.

Additionally, I do want the special case here - the legacy form will be removed in the future and I do not want to find a general solution to this specific problem (avoiding having to recheck _all_ of the uses of `form--column`)